compiler/src/typed/typemod.re

```diff
@@ -1081,6 +1092,13 @@ let report_error = ppf =>
       Includemod.report_error,
       errs,
     )
+  | Include_module_name_mismatch(provided_name, actual_name) =>
+    fprintf(
+      ppf,
+      "This statement includes module %s, but the file at the path defines module %s",
```

Comment:
Do you think they'd be trying to define an alias? Perhaps we should add "Did you mean to write `include %s as %s"?

Reply:
It's possible... I'll add it.